### DISCUSSION TITLE
Roland / Archibald's campaigns: "all are allied against you"

### DISCUSSION THREAD

**Discussion by a1exsh** [ihhub/fheroes2#4576] [General]
I've noticed that in a number of campaign scenarios the description says "all are allied against you", but that is not always the case: the AI heroes attack each other.

Has anyone played the campaign in the OG recently and can recall if the issue was also present there?  In any case it looks like a bug that should be fixed.

I will collect the maps that has the issue and update the list here:

- Archibald's campaign:
   - Evil 6: Rebellion / "Peasants!"
      - In the OG Editor there doesn't seem to be any special conditions, so it must be hard-coded:
        ![editor2_002](https://user-images.githubusercontent.com/489601/141682878-2e4a0be3-3dfd-4c72-ba97-ee317d333ea0.png)
   - Final scenario

**Comment by Branikolog** [comment]
Nice catch, @a1exsh !
In the OG enemy heroes are definitely allied (not sure about color with enemy Archibald/Roland, but those three, who in the central part of the map are never attacking each other)
